api/ring-device.ts

```diff
@@ -32,6 +32,12 @@ export class RingDevice {
     return this.data.name
   }
 
+  get onComponentDevices() {
+    return this.location.onDevices.pipe(
+      map((devices) => devices.filter(({ data }) => data.parentZid === this.id))
+    )
+  }
```

Comment:
Shouldn't need the getter here.  Can just create an observable directly as a member.  The nice thinng is that they don't actually execute unless they are subscribed to, so it's no additional load if you don't use a getter.

Reply:
Nice catch! Fixed in https://github.com/dgreif/ring/pull/331/commits/697fb3244b860642dd92f86aade3adaac3e5ec7f